personalize.module

```diff
@@ -1129,30 +1129,33 @@ function personalize_verify_agent($agent_name, $show_messages = TRUE) {
  * @param stdClass $agent_data
  *   (Optional) When provided then agent-specific transition steps can be
  *   included.  This is used for the "Scheduled" status.
+ * @param bool $include_all
+ *   (Optional) Include all transitions that are available.  If FALSE, only
+ *   those transitions that should be shown outside the campaign details
+ *   are returned.
```

Comment:
OK so my problem with this is that the distinction between transitions that should be included and those that should not when this parameter is set to FALSE doesn't come from personalize module - and so there's seemingly no logic around which ones are excluded. The distinction is only made in Acquia Lift module, so that logic belongs there. Can we remove this parameter from the function and just have Acquia Lift module unset the options it won't allow based on its notion of "changes editability status"?